### DISCUSSION TITLE
SlickGrid updates

### DISCUSSION THREAD

**Discussion by g-parki** [bokeh/bokeh#11685] [Internals and design (Q&A)]
I recently submitted a SlickGrid PR to resolve issue #11557. My PR's been merged already and included in release 2.4.41. What's the best way to integrate that change with our forked version? I can submit a duplicate PR to our fork if necessary...
cc @mattpap

**Comment by mattpap** [comment]
I updated the fork. Now this needs to be tested. You can accomplish this by cloning the fork (branch `bokeh@2.4.41`), pointing bokehjs' `package.json` to your local slickgrid (instead of npm repository), rebuilding bokehjs and eventually testinging it.

**Reply by g-parki** [reply]
Thanks, will do
